HackerRank/Algorithms/implementation/The_Time_in_Words.java

```diff
@@ -58,16 +64,19 @@ else if(m==45)
 		{
 			time="quarter to "+word[h+1];
 		}
-		else if(m<30)
+		else if(m<30 && m!=1)
 		{
 			time=word[m]+" minutes past "+word[h];
 		}
+        else if(m==1)
```

Comment:
lets swap this else if with line 67 so we don't need `m!=1 `

Reply:
The logic does not work then.

First, if m==1 then it will set the string to "word[m] minute past word[h]". Then once it reaches if(m<30), then it will RESET IT to "word[m] MINUTES past word[h]"

Reply:
I didn't get you here. Since its else if so only one of the statements will be executed so ideally `m==1` should just after `m==0` check so why would it reset it `word[m] MINUTES past word[h]` ?

Reply:
@EvilSourcerer seems like this is not fixed. Can you please fix it ?